index.js

```diff
@@ -202,9 +203,17 @@ ExtractTextPlugin.prototype.extract = function(options) {
 	options = mergeOptions({omit: before.length, remove: true}, options);
 	delete options.loader;
 	delete options.fallbackLoader;
-	return [this.loader(options)]
+	var ret = [this.loader(options)]
```

Comment:
In one place you write 'result' and there 'ret'. I think that first one if more verbose option 👍